ruby_event_store/lib/ruby_event_store/mappers/protobuf.rb

```diff
@@ -16,7 +16,6 @@ class Protobuf
       include PipelineMapper
 
       def initialize(events_class_remapping: {})
-        require_optional_dependency
```

Comment:
Not sure about this one, I'd rather like to see fail because of missing dependency before first use of mapper.

Reply:
Moved to https://github.com/RailsEventStore/rails_event_store/blob/1767bd70da4db24a39788833526c7d27decde79d/lib/ruby_event_store/mappers/protobuf_nested_struct_metadata_mapper.rb#L20

But that is still called in initalizer of `ProtobufNestedStructMetadataMapper` which is called in ctor of `Mappers::Protobuf`, and if you use that mapper in your client you initalize it when creating new client instance. For me that's before first use ;)